src/NodeVisitor/GlobalVariableVariableVisitor.php

```diff
@@ -30,7 +30,7 @@ public function enterNode(Node $node)
                 continue;
             }
 
-            $this->addContextMessage(
+            $this->addContextWarning(
                 'Complex variable without curly braces in global keyword',
```

Comment:
I think this should be an error because code like `global $$foo->bar;` produces a parse error in PHP 7